node/src/components/consensus/protocols/zug.rs

```diff
@@ -510,6 +518,10 @@ impl<C: Context + 'static> Zug<C> {
         if let Some(echo_map) = proposal_hash.and_then(|hash| round.echoes().get(&hash)) {
             echoes = self.validator_bit_field(first_validator_idx, echo_map.keys().cloned());
         }
+
+        // We register the node to which we asked the sync for:
```

Comment:
Again, we register the request ID, which has nothing to do with the node.

Reply:
Done